assets/blocks/single-page.js

```diff
@@ -9,9 +9,6 @@ import { registerCourseCompletedActionsBlock } from './course-completed-actions'
 
 const blocks = [ LearnerCoursesBlock, LearnerMessagesButtonBlock ];
 
-if ( window.sensei_single_page_blocks.course_completed_page_enabled ) {
-	registerCourseCompletedActionsBlock();
-	blocks.push( CourseResultsBlock );
-}
-
+registerCourseCompletedActionsBlock();
+blocks.push( CourseResultsBlock );
```

Comment:
I think now we can just add it directly to the blocks array, right?

Reply:
https://github.com/Automattic/sensei/pull/4280/commits/07e9cc6df2c17d8ce36552037eee904f708f256f